CHANGELOG.md

```diff
@@ -2,18 +2,20 @@
 
 ## Unreleased
 
-## `0.3.4` - 2020-03-25
+## Breaking Changes
 
-### Changed
+- The FPS Starter Project now requires Unity 2019.3 [#263](https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/263)
+
+## `0.3.4` - 2020-03-25
 
-- Upgraded to GDK for Unity version `0.3.4`
 ## Breaking Changes
 
 - The FPS starter project now uses [query-based interest](https://docs.improbable.io/reference/14.4/shared/authority-and-interest/interest/query-based-interest-qbi) instead of [chunk-based interest](https://docs.improbable.io/reference/14.4/shared/authority-and-interest/interest/chunk-based-interest-cbi). [#253](https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/253)
 - You must now pass in an `EntityId` to create a player `EntityTemplate`. [#254](https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/254)
 
 ## Changed
 
+- Upgraded to GDK for Unity version `0.3.4`
```

Comment:
We're doing this for version 0.3.5, right?

Reply:
Yup, this was just me tidying up the previous release's changelog. There were 2 changed sections for some reason

Reply:
looking at the diff I think it's just moving this from above breaking changes into the `changed` section